tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/basic/api/ScalarHolder.java

```diff
@@ -47,7 +48,7 @@
     private char charPrimitive;
     private Character charObject;
 
-    private char[] stringPrimitive;
+    private char[] charArray;
```

Comment:
This change also requires a change to the `basicScalar/inputgraphql` and `basicScalar/output.json` - changing `stringPrimitive` to `charArray`, with output like:
```
"charArray": [
        "1",
        "2",
        "3"
      ],
```

Reply:
Thanks. Fixed it. Will be included in the next commit